x-pack/legacy/plugins/infra/public/pages/metrics/components/chart_section_vis.tsx

```diff
@@ -42,15 +42,15 @@ export const ChartSectionVis = ({
   seriesOverrides,
   type,
 }: VisSectionProps) => {
-  if (!metric || !id) {
```

Comment:
:information_source: This condition violated the hook call order invariant so it was moved down beyond the hook calls.